apps/src/music/views/SoundStyle.module.scss

```diff
@@ -0,0 +1,108 @@
+@import 'color.scss';
+
+.sound {
+  &Beat {
```

Comment:
optional - I think you could use a mixin here to avoid duplicating the structure if you'd like? Something like
```
@mixin sound-type-style($type, $color, $fill, $background, $border) {
  &#{type} {
    &Color { color: $color }
    &Fill { fill: $fill }
    ...
  }

...

.sound {
  @include sound-type-style(Beat, #FC55FF, #FC55FF, ... );
  @include sound-type-style(Bass, ... );
  ...
}
```

Reply:
Love it.  Done.